Multisig/UI/Settings/Passcode/PasscodeViewController.swift

```diff
@@ -0,0 +1,144 @@
+//
+//  PasscodeViewController.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 2/22/21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class PasscodeViewController: UIViewController, UITextFieldDelegate {
+    @IBOutlet weak var scrollView: UIScrollView!
```

Comment:
Don't we need all of those outlets to be private?

Reply:
I wanted them to be accessible in the subclasses, so didn't add private